app/services/publishers/vacancies/vacancy_step_process.rb

```diff
@@ -20,6 +20,10 @@ def initialize(current_step, vacancy:, organisation:, session:)
     })
   end
 
+  def validatable_steps
+    steps - %i[job_location schools documents review]
```

Comment:
Why minus job_location and schools? I think these steps should be validated (if they are among `steps`).

Reply:
They were here before, I took them out, still works, good shout! 🏆